### DISCUSSION TITLE
Add the ability to "remove site" just like you can add a site for a specific entry

### DISCUSSION THREAD

**Discussion by jan-dh** [craftcms/cms#11406] [Ideas] [answered]
Currently you can add sites when you choose the section option:

<img width="410" alt="image" src="https://user-images.githubusercontent.com/5801781/172804637-43d2ca69-7c95-4a0f-b2ec-4a239d7f4662.png">

This by itself is pretty awesome and solves a lot of issues we had in a multi-site setup in the past.  However, to complete the circle, I would love to have the ability, once a site is added for an entry to also remove it in the future. Right now, once it's added you can only disable it after. Which can still lead to a lot of "ghost" pages with a disabled status. So once you "add" a site, also have the option to "remove a site"

<img width="363" alt="image" src="https://user-images.githubusercontent.com/5801781/172804512-8f467f1a-16ca-456b-bfc5-5e3110d62070.png">

**Comment by brandonkelly** [comment, marked as answer]
This is already possible, via the “Delete entry for this site” option in the **Save** button’s alternative action menu:

<img alt="The Save button menu with “Delete entry for this site” highlighted" src="https://user-images.githubusercontent.com/47792/172959503-ecb538d2-7e9e-4859-a85b-ab8af30fa862.png" width="281" height="286">
